app/models/preprint-provider.ts

```diff
@@ -52,9 +53,15 @@ export default class PreprintProviderModel extends ProviderModel {
         };
     }
 
+    @computed('documentType.plural')
+    get searchPlaceholder(): string {
+        return this.intl.t('preprints.header.search_placeholder',
+            { placeholder: this.documentType.plural});
+    }
+
     @computed('id')
     get preprintWordInTitle() {
```

Comment:
Super-minor: this might should change to 'preprintWordNotInTitle' or similar to reflect the change in logic.